index.html

```diff
@@ -1,12 +1,131 @@
+      <ol class="bloks">
+        <li class="block1">
+          <h2>Настоящий городок</h2>
+          <p>Седона — не аттракцион<br> для туристов, там течёт<br>своя жизнь</p>
+          <img src="" alt="Фасад домов.">
+        </li>
+        <ul class="miniblock">
+          <li class="mini1">
+            <img src="" alt="Дом.">
+            <h3>Жильё</h3>
+            <p>Рекомендуем пожить<br>в настоящем мотеле,<br>всё как в кино!</p>
+          </li>
+          <li class="mini2">
+            <img src="" alt="Бургер.">
+            <h3>Еда</h3>
+            <p>Всегда заказывайте<br>топовый фирменный бургер,<br> вы не разочаруетесь!</p>
+          </li>
+          <li class="mini3">
+            <img scr="" alt="Перевязанная коробка">
+            <h3>Сувениры</h3>
+            <p>Не только китайского,<br>но и настоящего местного<br>производства!</p>
+          </li>
+        </ul>
+        <li class="block2">
+          <img src="" alt="Скалы каньона"
+          <h2>Там есть<br>мост дьявола</h2>
+          <p>Да, по нему можно пройти!<br> Если вы осмелитесь,<br> разумеется</p>
+        </li>
+        <li class="block3">
+          <h2>Небольшая<br>площадь</h2>
+          <p>Все достопримечательности<br>находятся очень близко</p>
+        </li>
+        <li class="block4">
+          <h2>Красивая<br>дорога</h2>
+          <p>Ехать в Седону из Лас-Вегаса<br>совсем не скучно!</p>
+        </li>
+        <li class="block5">
+          <h2>Мало туристов</h2>
+          <p>Большинство едет в Гранд Каньон<br>и толпится там</p>
+        </li>
+      </ol>
+    </section>
+
+    <h2>Заинтересовались?</h2>
+    <p>Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+    <form class="appointment-form" action="https://echo.htmlacademy.ru/" method="post">
+      <button class="button order-button" type="submit">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</button>
+    </form>
+
+  </main>
+  <footer class="page-footer">
+    <h2>Подпишитесь на рассылку</h2>
+    <p>Только полезная информация и никакого спама,<br>честное бойскаутское!</p>
+    <form class="appointment-form" action="https://echo.htmlacademy.ru/" method="post"></form>
+    <label for="e-mail">
+      <input id="e-mail" type="text" placeholder="Ваш e-mail">
+    </label>
+    <button class="button" type="submit">Подписаться</button>
+
+    <ul class="footer-social-list">
+      <li class="footer-social-item">
+        <a class="button" href="https://www.fasebook.com/htmlacademy">
+          <span class="visually-hidden">Facebook</span>
+        </a>
+      </li>
+      <li class="footer-social-item">
+        <a class="button" href="https://twitter.com/htmlacademy_ru">
+          <span class="visually-hidden">Twitter</span>
+        </a>
+      </li>
+      <li class="footer-social-item">
+        <a class="button" href="https://www.youtube.com/c/HTMLAcademyRUS">
+          <span class="visually-hidden">Youtube</span>i
```

Comment:
```suggestion
          <span class="visually-hidden">Youtube</span>
```

лишнее затесалось, внимательно проверяй свой код